src/components/CodeReminderModal/index.jsx

```diff
@@ -187,12 +184,13 @@ class CodeReminderModal extends React.Component {
   }
 
   renderTitle() {
-    const { title } = this.props;
-
+    const { title, data } = this.props;
     return (
       <React.Fragment>
         <span className="d-block">{title}</span>
-        <small>Assignment Reminder</small>
+        <small>
+          {data.selectedCodes && data.selectedCodes.length === 0 ? 'Remind All' : 'Assignment Reminder'}
```

Comment:
Similar to the comment above, I think this is something we should do for the other modals (i.e., assign, revoke) as well.